client/signup/main.jsx

```diff
@@ -298,6 +301,11 @@ class Signup extends React.Component {
 		}
 	};
 
+	preloadNextStep( stepName ) {
+		const nextStep = flows.getNextStepNameInFlow( this.props.flowName, stepName );
+		nextStep && asyncLoadStep( nextStep );
```

Comment:
`asyncLoadStep` expects a file name, not a step name, so I tested this PR by editing the following:

```
import { getStepComponent, asyncLoadStep, stepNameToFilenameMap } from './config/step-components';
....

nextStep && asyncLoadStep( get( stepNameToFilenameMap, nextStep ) );
```

Works well! The steps load and, at least for me, works pretty well. ❤️ I throttled to 3G and it was a marked improvement.

I'm not sure how to square the above with @jsnajdr 's [comments](https://github.com/Automattic/wp-calypso/pull/32718/files#r280109607), but once we work out how to pass the file name vs the step name between these three methods, then I think it'll work nicely.